lib/Doctrine/Record/Filter/Compound.php

```diff
@@ -32,38 +32,48 @@
  */
 class Doctrine_Record_Filter_Compound extends Doctrine_Record_Filter
 {
+    /**
+     * @var string[]
+     */
     protected $_aliases = array();
 
+    /**
+     * @param string[] $aliases A list of relation name
+     */
     public function __construct(array $aliases)
     {
         $this->_aliases = $aliases;
     }
 
+    /**
+     * @throws Doctrine_Table_Exception
+     */
     public function init()
     {
-    	// check that all aliases exist
-    	foreach ($this->_aliases as $alias) {
+        // check that all aliases exist
+        foreach ($this->_aliases as $alias) {
             $this->_table->getRelation($alias);
-    	}
+        }
     }
 
     /**
-     * filterSet
-     * defines an implementation for filtering the set() method of Doctrine_Record
+     * Provides fallback for setting value.
```

Comment:
we should have a more detailed comment here
Do we have a better documentation for this method @alquerci ?

Reply:
What information are missing ?

Reply:
Either we use the parent's documentation, or we remove the `fallback` as it is not clear to which fallback it is referring IMO.

Reply:
Indeed, the fallback is related to its usage, like the current comment.

https://github.com/FriendsOfSymfony1/doctrine1/blob/master/lib/Doctrine/Record.php#L1503

_from Clean Code book_
> All too often the comments get separated from the code they describe and become orphaned blurbs of ever-decreasing accuracy.

I think this case it is. That's why it is not clear.

---

So, I agree to remove the `fallback` wording, it makes sense.

b5b0ce1

wdyt?